client/blocks/product-purchase-features-list/index.jsx

```diff
@@ -94,10 +96,25 @@ export class ProductPurchaseFeaturesList extends Component {
 		const {
 			isPlaceholder,
 			plan,
+			currentPlan,
 			planHasDomainCredit,
 			selectedSite,
 			showCustomizerFeature,
+			hasAvailableConciergeSessions,
 		} = this.props;
+
+		let isBusinessOnboardingAvailable = false;
+
+		// TODO: remove this and just have isBusinessOnboardingAvailable = hasAvailableConciergeSessions once free QS sessions are actually removed from Business Plan
+		if ( currentPlan && hasAvailableConciergeSessions ) {
```

Comment:
Do you know if `hasAvailableConciergeSessions` will also be true if the user has purchased a concierge session? If it does then it seems like this will also hide even a purchased session.
I wonder if the value returned by `getHasAvailableConciergeSessions()` is what should be changed.
cc @niranjan-uma-shankar since he has experience with these parts.

Reply:
`hasAvailableConciergeSessions` is used to determine if the user has any concierge session available - included or purchased. So yes, it will be true if the user has an _unused_ purchased concierge session.

If I understand correctly, if a user has purchased a session on a Business plan, then this section will show for existing Business plan users, am I right? They would be taken to the scheduler and that seems about right, but are you preferring to hide the card?

Reply:
@niranjan-uma-shankar , the card for the included session should be hidden for new Business plan purchases (it's no longer included in Business). But I would think that we'd still want to continue showing the card if a user has purchased a session. Wouldn't this logic result in the card being hidden once the plan expiry is after `2021-07-31` even if the site has an unused purchased session? It's possible I've misunderstood something too.

Do you think it would make sense to update the endpoint and check for the expiry date there instead of here in the front-end?

Reply:
@travisw @niranjan-uma-shankar 
I replaced `hasAvailableConciergeSessions` with checking for `scheduleId`. Now the `Get personalised help` section should appear if:
- plan is old (expires before 2021-07-31) and has included sessions: `scheduleId === 1`
- user has purchased a QS session for that plan (new or old): `scheduleId > 1`, to include the 2 and any overrides via the `get_eligible_schedule_id` hook )